deployment/ansible/group_vars/all

```diff
@@ -14,6 +14,10 @@ postgresql_username: rf
 postgresql_password: rf
 postgresql_database: rf
 
+aws_key: ""
```

Comment:
It might be easier to [setup](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration) AWS credentials under a Raster Foundry specific profile using the AWS CLI on your workstation, then mount that in to `~/.aws/profile` of whatever user is going to be making the API calls. That'll allow a little bit more seamless local development. Also, in production none of that will matter because we'll be leaning on IAM roles.

(This depends on us using an AWS SDK to sign the URL.)